language/pl-PL/com_users.ini

```
@@ -4,119 +4,159 @@
 COM_USERS_REGISTRATION_ACTIVATION_SAVE_FAILED="Błąd podczas zapisywania danych aktywacyjnych: %s"
-COM_USERS_REGISTRATION_ADMINACTIVATE_SUCCESS="Konto użytkownika zostało pomyślnie zatwierdzone, a użytkownik został o tym poinformowany."
-COM_USERS_REGISTRATION_COMPLETE_ACTIVATE="Twoje konto zostało utworzone i odnośnik aktywacyjny został wysłany na wprowadzony adres e-mail. Kolejnym krokiem jest konieczność aktywacji konta, uczynisz to, klikając odnośnik aktywacyjny w wiadomości e-mail, którą dostaniesz. Aktywacja konta jest wymagana przed pierwszym logowaniem na stronie."
-COM_USERS_REGISTRATION_COMPLETE_VERIFY="Twoje konto zostało utworzone i odnośnik weryfikacyjny został wysłany na wprowadzony adres e-mail. Kolejnym krokiem jest konieczność weryfikacji konta, uczynisz to, klikając odnośnik weryfikacyjny w wiadomości e-mail, którą dostaniesz. Po pomyślnej weryfikacji Administrator zatwierdzi Twoje konto przed  pierwszym logowaniem."
+COM_USERS_REGISTRATION_ADMINACTIVATE_SUCCESS="Konto użytkownika zostało pomyślnie aktywowane i użytkownik został o tym powiadomiony."
+COM_USERS_REGISTRATION_COMPLETE_ACTIVATE="Twoje konto zostało utworzone i odnośnik aktywacyjny został wysłany na podany adres e-mail. Pamiętaj, że musisz aktywować konto, wybierając odnośnik aktywacyjny w otrzymanej wiadomości e-mail, którą dostaniesz. Aktywacja konta jest wymagana przed pierwszym zalogowaniem się na stronie internetowej."
+COM_USERS_REGISTRATION_COMPLETE_VERIFY="Twoje konto zostało utworzone i odnośnik aktywacyjny został wysłany na podany adres e-mail. Pamiętaj, że musisz aktywować konto, wybierając odnośnik aktywacyjny w otrzymanej wiadomości e-mail, którą dostaniesz. Aktywacja konta i weryfikacja przez administratora jest wymagane przed pierwszym zalogowaniem się na stronie internetowej."
 COM_USERS_REGISTRATION_DEFAULT_LABEL="Rejestracja użytkownika"
 COM_USERS_REGISTRATION_SAVE_FAILED="Błąd podczas rejestracji: %s"
-COM_USERS_REGISTRATION_SAVE_SUCCESS="Dziękujemy za rejestrację. Możesz teraz zalogować się na stronie przy pomocy nazwy użytkownika i hasła użytych podczas rejestracji."
-COM_USERS_REGISTRATION_SEND_MAIL_FAILED="Wystąpił błąd podczas wysyłania wiadomości e-mail w czasie rejestracji użytkownika. Wiadomość została wysłana do Administratora tej strony."
-COM_USERS_REGISTRATION_VERIFY_SUCCESS="Twój adres e-mail został zweryfikowany. Kiedy Administrator zatwierdzi Twoje konto, zostanie wysłany do Ciebie e-mail z informacją, po otrzymaniu tej wiadomości będzie można już zalogować się na stronie."
+COM_USERS_REGISTRATION_SAVE_SUCCESS="Dziękujemy za rejestrację. Możesz teraz zalogować się na stronie internetowej przy pomocy nazwy użytkownika i hasła użytych podczas rejestracji."
+COM_USERS_REGISTRATION_SEND_MAIL_FAILED="Wystąpił błąd podczas wysyłania wiadomości e-mail w czasie rejestracji użytkownika. Wiadomość została wysłana do administratora tej strony internetowej."
```

Skoro już poprawiamy, to "stronę internetową" zamieniałbym konsekwentnie na "witrynę"
```suggestion
COM_USERS_REGISTRATION_SEND_MAIL_FAILED="Wystąpił błąd podczas wysyłania wiadomości e-mail w czasie rejestracji użytkownika. Wiadomość została wysłana do administratora tej witryny."
```